CHANGELOG.md

```diff
@@ -4,6 +4,67 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.21.0 (2025-07-08)
+
+This release contains performance improvements and bug fixes since the 2.20.3 release. We recommend that you upgrade at the next available opportunity.
+
+**Highlighted features in TimescaleDB v2.21.0**
+* The attach & detach chunks feature allows manually adding or removing chunks from a hypertable with uncompressed chunks, similar to PostgreSQL’s partition management.
+* Continued improvement of backfilling into the columnstore, achieving up to 2.5x speedup for constrained tables, by introducing caching logic that boosts throughput for writes to compressed chunks, bringing `INSERT` performance close to that of uncompressed chunks.
+* Optimized `DELETE` operations on the columstore through batch-level deletions of non-segmentby keys in the filter condition, greatly improving performance to up to 42x faster in some cases, as well as reducing bloat, and lowering resource usage.
+* The locking mechanism for Continuous Aggregate refresh policies was removed, enabling concurrent refreshes for non-overlapping ranges and eliminating the need for complex customer workarounds.
```

Comment:
```suggestion
* The heavy lock taken in Continuous Aggregate refresh was relaxed, enabling concurrent refreshes for non-overlapping ranges and eliminating the need for complex customer workarounds.
```
What I want to say here is that not only the refresh policy get rid of heavy locks, but also a manual refresh running the `refresh_continuous_aggregate` procedure (that is used for customers workarounds).